### PR TITLE
Backport changes for productized camel k in 1.6.x

### PR DESCRIPTION
**Pull request by nicolaferraro**

### CODE REVIEW

**Comment by heiko-braun** [comment]
@orpiske Mind a review?

**Comment by heiko-braun** [comment]
@nicolaferraro Is all of this present on master as well?

**Comment by nicolaferraro** [comment]
Master miss only https://github.com/syndesisio/fuse-online-install/pull/103.

I don't know why, but master is configured to work on a released version of Syndesis (1.2) using official repos. So you have to flip some config in the `fuse_online_config.sh` to make it work. The 1.6.x branch (after this change) should be aligned to what we're building in brew.

**Comment by heiko-braun** [comment]
We need to clean up master to be the latest, i.e. containg everything that's currenlty on 1.6.x

File: fuse_online_config.sh [hunk 1]
```diff
@@ -19,6 +20,8 @@ repository="fuse7"
 
 # Test:
 registry="brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888"
+maven_repository="https://origin-repository.jboss.org/nexus/content/groups/ea@id=redhat.ea"
```
[the remark this answers is not in the document]
Comment (vhalbert) [reply fuse_online_config.sh]: yes, will make this property work the same way registry is handled.   Will need to push a change to the config template.

**Comment by orpiske** [comment]
> @orpiske Mind a review?

I bit too late already, but I just wanted to say that it looks good to me.

**Pull request merged**